src/Microsoft.Azure.WebJobs.Host/Executors/FunctionExecutor.cs

```diff
@@ -153,6 +159,19 @@ public HostOutputMessage HostOutputMessage
             return exceptionInfo != null ? new ExceptionDispatchInfoDelayedException(exceptionInfo) : null;
         }
 
+        private async Task InvokeExceptionFiltersAsync(Exception exception, IFunctionInstance functionInstance, CancellationToken cancellationToken)
+        {
+            var exceptionFilters = GetFilters<IFunctionExceptionFilter>(_extensionRegistry, functionInstance.FunctionDescriptor, functionInstance.JobInstance);
+            if (exceptionFilters.Any())
+            {
+                var exceptionContext = new FunctionExceptionContext(functionInstance.Id, functionInstance.FunctionDescriptor.ShortName, _logger, exception);
+                foreach (var exceptionFilter in exceptionFilters)
+                {
+                    await exceptionFilter.OnExceptionAsync(exceptionContext, cancellationToken);
```

Comment:
Now handling this the same way as we do for invocation filters. We keep track of the last exception, keep executing filters, and the last exception wins